test/widgets/action_sheet_test.dart

```diff
@@ -442,6 +445,44 @@ void main() {
             });
       });
 
+      testWidgets('on topic move, acts on new topic', (tester) async {
+        final stream = eg.stream();
+        const topic = 'old topic';
+        final message = eg.streamMessage(flags: [MessageFlag.read],
+          stream: stream, topic: topic);
+        await setupToMessageActionSheet(tester, message: message, narrow: TopicNarrow.ofMessage(message));
+        // Get the action sheet fully deployed while the old narrow applies.
+        await tester.pumpAndSettle();
```

Comment:
Hmm, the test passes and would fail as expected without the fix, if I remove this. I see that `setupToMessageActionSheet` calls `tester.pump` with a 250ms delay. Maybe that's sufficient?

Reply:
I'll expand the comment. Does this version explain why this call is here?
```dart
        // Get the action sheet fully deployed while the old narrow applies.
        // (This way we maximize the range of potential bugs this test can catch,
        // by giving the code maximum opportunity to latch onto the old topic.)
        await tester.pumpAndSettle();
```